andela/ah-fulldeck-frontend
# #162948946 setup react boilerplate

#### What does this PR do?
* It sets up an empty React project using Airbnb linting option.

#### Description of Task to be completed?
* Simply use `npm create-react-app .` on the existing repository.
* Configer `eslint`
#### How should this be manually tested?
* Clone the github repository `git clone https://github.com/andela/ah-fulldeck-frontend` on your local machine.
* Change your working directory to the project folder `cd ah-fulldeck-frontend`
* Install the required dependencies using `npm install`
* Run the local development server using `npm start` and you should get the following message:
  ```
  Compiled successfully!
  
  You can now view ah-fulldeck-frontend in the browser.
  
    Local:            http://localhost:3000/
    On Your Network:  http://10.10.160.30:3000/
  
  Note that the development build is not optimized.
  To create a production build, use npm run build
  ```

#### Any background context you want to provide?
Create React App now supports the use of different environment variables files out of the box. For
more information click on this link: [here](https://medium.com/@tacomanator/environments-with-create-react-app-7b645312c09d).

Therefore, other than creating the relevant files, nothing more needs to be done to configure their support.

#### What are the relevant pivotal tracker stories?
[#162948946](https://www.pivotaltracker.com/story/show/162948946)

#### Screenshots (if appropriate)
N/A
